dsDisplay.c

```diff
@@ -30,12 +30,27 @@
 #include "dsVideoResolutionSettings.h"
 #include "dshalUtils.h"
 
+
 #define MAX_HDMI_CODE_ID (127)
 dsDisplayEventCallback_t _halcallback = NULL;
 dsVideoPortResolution_t *HdmiSupportedResolution=NULL;
 static unsigned int numSupportedResn = 0;
 static bool _bDisplayInited = false;
 static bool isBootup = true;
+
+static hdmiSupportedRes_t resolutionMap[] = {
```

Comment:
It would make sense to add this in a header file to avoid multiple copies.
That will ensure there is no mismatch when getting updated in future.
Can this be moved to `dshalUtils.h` ?